### PR TITLE
Cypress scenarios for setting up OSSM Console

### PR DESCRIPTION
**Pull request by ScriptingShrimp**
This PR adds hack script to manage OSSMC life cycle. Feature file contain scenarios annotated with `@install` tag, which will be executed as first script via npm/yarn. Steps implementation is straight forward, calling hack script by `cy.exec()`.

Developed and tested against OSSM 2.5 and OCP 4.13.

### CODE REVIEW

**Comment by jmazzitelli** [comment]
I believe this does what you want already : https://github.com/kiali/kiali/blob/master/hack/install-kiali-ossmc-openshift.sh

**Comment by ScriptingShrimp** [comment]
> I believe this does what you want already : https://github.com/kiali/kiali/blob/master/hack/install-kiali-ossmc-openshift.sh

I checked the scripts, currently I will leave as it is (time concerns).  I'm not opposed to refactor this and consolidate into kiali/kiali and copy into this repo by `hack/copy-frontend-src-to-ossmc.sh`. Shall we file an issue?

File: hack/manage-ossmc.sh [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+-h|--help
+    Print this help message.
+-i|--install
+    Instruct the Kiali Operator to create a small OSSMConsole CR.
+-u|--uninstall 
+    Remove the OSSMConsole CR from all namespaces.
+-v|--validate-ossmc 
+    Check the OSSM Console plugin via Kiali script.
+-w|--wait-ossmc
+    Wait for the OSSMC given pod to be ready.
```
Comment (jmazzitelli) [inline hack/manage-ossmc.sh:51]:
```suggestion
    Wait for the OSSMC plugin pod to be ready. 
```

File: hack/manage-ossmc.sh [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    exit 255
+fi
+
+while [[ $# -gt 0 ]]; do
+  key="$1"
+  case $key in
+    -h|--help)                  helpmsg;                        exit $?; ;;
+    -i|--install)               install;                        exit $?; ;;
+    -u|--uninstall)             uninstall;                      exit $?; ;;
+    -v|--validate-ossmc)         validate-ossmc;                     exit $?; ;;
```
Comment (jmazzitelli) [inline hack/manage-ossmc.sh:67]: trivial... spacing:

```suggestion
    -v|--validate-ossmc)        validate-ossmc;                exit $?; ;;
```

**Review by jmazzitelli — CHANGES_REQUESTED** [review]

**Review by jmazzitelli — APPROVED** [review]
I didn't run it, but the script looks good.

**Review by ferhoyos — APPROVED** [review]
LGTM, tested locally and works fine.

**Pull request merged**